src/test/java/com/hillel/items_exchange/controller/UserControllerTest.java

```diff
@@ -133,4 +158,94 @@ private ResultActions getResultActions(HttpMethod httpMethod, String path,
+
+    }
+
+    @Test
+    @WithMockUser(username = "admin")
+    @Transactional
+    @DataSet("database_init.yml")
+    @ExpectedDataSet(value = "/children/create.yml", ignoreCols = {"birth_date", "sex"})
+    void addChildren_Success_ShouldReturnHttpStatusOk() throws Exception {
+        mockMvc.perform(post("/user/child/")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(validCreatingChildDtoJson)
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isOk());
+    }
+
+    @Test
+    @WithMockUser(username = "admin")
+    @Transactional
+    @DataSet("database_init.yml")
+    @ExpectedDataSet(value = "children/delete.yml")
+    void removeChild_Success_ShouldReturnHttpStatusOk() throws Exception {
+        mockMvc.perform(delete("/user/child/1")
+                .contentType(MediaType.APPLICATION_JSON)
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isOk());
+    }
+
+    @Test
+    @WithMockUser(username = "admin")
+    @Transactional
+    @DataSet("database_init.yml")
+    @ExpectedDataSet(value = "children/update.yml")
+    void updateChild_Success_ShouldReturnHttpStatusOk() throws Exception {
+        mockMvc.perform(put("/user/child/")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(validUpdatingChildDtoJson)
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isOk());
+    }
+
+    @Test
+    @WithMockUser(username = "admin")
+    @DataSet("database_init.yml")
+    void addChild_NotValidDto_ShouldThrowIllegalIdentifierException() throws Exception {
+        final MvcResult mvcResult = mockMvc.perform(post("/user/child/")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(notValidCreatingChildDtoJson)
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isBadRequest())
+                .andReturn();
+        assertTrue(mvcResult.getResponse().getContentAsString().contains("Zero ID is expected"));
+    }
+
+    @Test
+    @WithMockUser(username = "admin")
+    @DataSet("database_init.yml")
+    void updateChild_NotValidDto_ShouldThrowIllegalIdentifierException() throws Exception {
+        final MvcResult mvcResult = mockMvc.perform(put("/user/child/")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(notValidUpdatingChildDtoJson)
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isBadRequest())
+                .andReturn();
+        assertTrue(mvcResult.getResponse().getContentAsString().contains("Not all children from dto present in"));
+    }
+
+    private ChildDto childDtoBuilder(long id, LocalDate birthDate, String sex){
```

Comment:
According to the naming convention "Methods should be verbs", so I think you should rename this method